Source/OIDLogoutResponse.m

```diff
@@ -0,0 +1,84 @@
+/*! @file OIDLogoutResponse.m
+    @brief AppAuth iOS SDK
+    @copyright
+        Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Update all instances `2015 Google Inc. All Rights Reserved.` -> `2017 The AppAuth Authors.`

Reply:
Will do, thanks!